[Patch] without this patch you need to install libproc-processtable-perl at host system

--- a/v4l/Makefile
+++ b/v4l/Makefile
@@ -51,7 +51,7 @@
 	@echo Kernel build directory is $(OUTDIR)
 	$(MAKE) -C ../linux apply_patches
 	$(MAKE) -C $(OUTDIR) SUBDIRS=$(PWD) $(MYCFLAGS) modules
-	./scripts/rmmod.pl check
+#	./scripts/rmmod.pl check
 #	$(MAKE) checkpatch
 
 mismatch:: prepare firmware
